.github/workflows/build-image.yml

```diff
@@ -0,0 +1,32 @@
+# Triggered by Pull Request or Manually (from GitHub UI) events
+
+name: Publish image to Registry
+
+on:
+  pull_request:
```

Comment:
I think we need to build/push the image on merges to master as well

Reply:
ok